ngx-fudis/projects/ngx-fudis/src/lib/directives/link/link.directive.stories.ts

```diff
@@ -156,7 +126,7 @@ export const AllCombinations: StoryFn = (args) => ({
 Example.args = {
   title: 'This is an example title',
   external: false,
```

Comment:
Not an issue of this specific PR, but at some point between `3.2.1` and `4.0.0-rc.0` this story's `external` toggle has broke. It now needs "remount" the view to show the external icon when toggled from the controls.